pyintelowl/cli/analyse.py

```diff
@@ -145,3 +145,34 @@ def file(
         )
     except IntelOwlClientException as e:
         ctx.obj.logger.fatal(str(e))
+
+
+@analyse.command(help="Send multiple analysis requests from file")
```

Comment:
Change to `Send multiple analysis requests. Reads file (csv or json) for inputs.`

Reply:
So, `CSV` is to be supported?

Reply:
yes, didn't you get my message on the slack channel ?

Reply:
Got it.